src/models/User.js

```diff
@@ -26,13 +27,7 @@ const userSchema = mongoose.Schema({
         type: Boolean,
         default: false
     },
-    tokens: [{
-        token: {
-            type: String,
-            required: true
-        }
-    }]
-
+    invokedTokens: [ String ]
```

Comment:
The name `invokedTokens` suggests that it stores tokens not jti.
Change it to `invokedTokensId`